examples/basic/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "private": "true",
+  "scripts": {
+    "start": "react-scripts start"
+  },
+  "dependencies": {
+    "@razorpay/blade": "../../packages/blade/build/",
+    "@fontsource/lato": "^4.5.10",
+    "@types/react": "^18.0.25",
+    "@types/react-dom": "^18.0.9",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "^5.0.1",
+    "styled-components": "^5.3.6"
```

Comment:
I was thinking we can let this be since nothing major will break. Also this will test out the worst case scenarios. And the libraries are mostly widely used ones so don't see them breaking semver at any point.